packages/sdk/src/batchChangeMinters/README.md

```diff
@@ -2,9 +2,11 @@
 
 # Batch change minters
 
-Batch change minters for a smart contract you own by providing an array of the ids of the minters you would like to add and/or remove
+Change minting permissions for your contract by removing or adding multiple accountIds in one call.
 
-**As with all new SDK api methods, this call should be wrapped in [execute](../#execute) and passed a signing method
+AccountIds in the `removeMinters` array will lose minting permission for the specified contract. Ids in the `addMinters` array get granted that permission.
```

Comment:
AccountIds -> Account IDs, Ids -> IDs

What happens when the same ID shows up in both arrays?